packages/open-api-spec/README.md

```diff
@@ -0,0 +1,42 @@
+# @loopback/open-api-spec
+
+TypeScript type definitions for OpenAPI Spec/Swagger documents.
+
+## Overview
+
+TypeScript definitions describing the schema of OpenAPI/Swagger documents,
+including LoopBack-specific extensions.
+
+## Installation
+
+```
+$ npm install --save @loopback/open-api-spec
+```
+
+## Basic use
```

Comment:
Nope, see http://loopback.io/doc/en/contrib/README-guidelines.html#basic-use

> Basic use
> Tip: This section is often titled “Usage,” but we prefer “Use” because it is a shorter and more common word, and means the same thing.